### DISCUSSION TITLE
About ISCE2 and Dolphin

### DISCUSSION THREAD

**Discussion by Arcyzr** [isce-framework/dolphin#161] [General]
Hello, developers,

I am a novice in InSAR data processing. I have carefully reviewed the detailed introduction about "Dolphin" but still have some doubts. I am wondering if I can utilize the commands from ISCE2 (for example: stackSentinel.py -s ../SLC/ -d ../DEM/demLat_N30_N31_Lon_E103_E104.dem.wgs84 -a ../AuxDir/ -o ../Orbits -b '30 31 103 104' -c 6 -W slc) to generate the SLC required for "Dolphin," and then process it in "Dolphin" followed by conducting time-series analysis in "MintPy."

**Comment by scottstanie** [comment]
Yes this should be possible. I've only done one tests on an isce2 stripmap stack, but `dolphin` should be agnostic to the input processor. It will work on any coregistered stack of SLCs.

There may be extra work to get it into Mintpy if you're using radar coordinates, but I believe it should be adaptable from one of the existing `prep_` scripts if the required metadata is somewhere.
